gensim/summarization/mz_entropy.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+
+from gensim.summarization.textcleaner import tokenize_by_word as _tokenize_by_word
+from gensim.utils import to_unicode
+import numpy
+import scipy
+
+def mz_keywords(text,blocksize=1024,scores=False,split=False,weighted=True,threshold=0.0):
+    """Extract keywords from text using the Montemurro and Zanette entropy algorithm.
```

Comment:
Please use [numpy-style format](https://github.com/RaRe-Technologies/gensim/wiki/Developer-page#docstrings) for docstring (here and everywhere)

Reply:
Given that the return type varies according to the scores and split arguments, what should I put in the Returns section?

Reply:
enumerate all + add description what's condition for each type

Reply:
Can you point me to an example in the existing docs?

Reply:
@PeteBleackley sorry, but I have no example for this case,